ampl-converter/src/main/java/com/powsybl/ampl/converter/NetworkApplier.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.ampl.converter;
+
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.network.*;
+
+/**
+ * Interface to modify a network after an Ampl solve.<br>
+ * {@link AmplNetworkReader} does the Ampl output parsing,
+ * and {@link NetworkApplier} modifies the {@link Network}.<br>
+ * Also provides some utility functions for implementations :
+ * <ul>
+ *     <li>{@link NetworkApplier#busConnection}</li>
+ *     <li>{@link NetworkApplier#getThreeWindingsTransformerLeg}</li>
+ *     <li>{@link NetworkApplier#getThreeWindingsTransformer}</li>
+ * </ul>
+ * Default implementation in {@link DefaultNetworkApplier}.
+ *
+ * @author Nicolas Pierre <nicolas.pierre@artelys.com>
+ * @see AmplNetworkReader
+ * @see DefaultNetworkApplier
+ */
+public interface NetworkApplier {
```

Comment:
AmplNetworkUpdater would be a better name. I think "apply" is not meaningful enough.